mozetl/bhr_collection/bhr_collection.py

```diff
@@ -334,17 +349,14 @@ def ingest_row(self, row):
                 # If we're below the acceptance threshold, just lump it under (other) below
                 # its parent.
                 last_stack = stack_table.key_to_index(("(other)", lib_name, last_stack))
+                last_cache_item_index = cache_item_index
```

Comment:
What's this for?  `last_cache_item_index` doesn't seem to be used after the break

Reply:
Hmm, I think this snuck in, and isn't properly related to my annotations patch. I'll clean it up.